.vale.ini

```diff
@@ -1,31 +1,31 @@
-# Core settings
-# =============
-
-# Location of our `styles`
-StylesPath = "styles"
-
-# The options are `suggestion`, `warning`, or `error` (defaults to “warning”).
-MinAlertLevel = warning
-
-# By default, `code` and `tt` are ignored.
-IgnoredScopes = code, tt
-
-# By default, `script`, `style`, `pre`, and `figure` are ignored.
-SkippedScopes = script, style, pre, figure
-
-# WordTemplate specifies what Vale will consider to be an individual word.
-WordTemplate = \b(?:%s)\b
-
-# List of Packages to be used for our guidelines
-Packages = Google
-
-# Define the Ansys vocabulary
-Vocab = ANSYS
-
-[*.{md,rst}]
-
-# Apply the following styles
-BasedOnStyles = Vale, Google
-
-# Removing Google-specific rule - Not applicable under some circumstances
-Google.Colons = NO
+# Core settings
+# =============
+
+# Location of our `styles`
+StylesPath = "./doc/styles"
```

Comment:
The documentation build process working directory is `/doc` hence this was correct. I believe.

Reply:
I have tried different ways to fix it but I agree that this one didn't have an effect. I will remove it.